Archives/ClmFile.cpp

```diff
@@ -463,97 +458,94 @@ namespace Archives
 	}
 
 	bool ClmFile::WriteVolume(HANDLE outFile,
-		int numFilesToPack,
 		HANDLE *fileHandle,
 		IndexEntry *entry,
-		std::vector<std::string> internalNames,
+		const std::vector<std::string>& internalNames,
 		WaveFormatEx *waveFormat)
 	{
 #pragma pack(push, 1)
-		struct SClmHeader
+		struct ClmHeader
 		{
-			char textBuff[32];
+			ClmHeader() { memcpy(fileVersion.data(), "OP2 Clump File Version 1.0\x01A\0\0\0\0", 32); }
```

Comment:
Hmm, the 2 constructor option is probably the best idea. That should allow flexibility and efficiency. The Reset option seems like it would be a bit awkward in practice.

I'm also fine with ignoring this concern for now. If there's no problem, there's really nothing to fix. I think my bigger concern here, is how this might affect long term maintainability. Your two constructor proposal is probably the best there, though the currently solution is probably just fine.